metrics_utility/automation_controller_billing/collectors.py

```diff
@@ -207,6 +212,14 @@ def yaml_and_json_parsing_functions():
 
 @register('job_host_summary', '1.2', format='csv', description=_('Data for billing'), fnc_slicing=daily_slicing)
 def job_host_summary_table(since, full_path, until, **kwargs):
+    disable_job_host_summary_str = os.environ.get('METRICS_UTILITY_DISABLE_JOB_HOST_SUMMARY_COLLECTOR', 'false')
```

Comment:
This should be communicated with the team. We have METRICS_UTILITY_OPTIONAL_COLLECTORS for this.

And we must double check that disabling job_host_summary will work. Maybe we have some requirements that it is mandatory.

Im preparing PR that allows to gather 0 days (thus only snapshots). We may try both ways. Setting different value than 4 weeks can be valuable not only in this scenario.